### PR TITLE
modifications to support liconn data

### PR DESCRIPTION
**Pull request by gbhuang**
this pull request incorporates two feature changes while skipping recent modifications in master:

- normalization allows for specifying of min and max input range, as alternative to 0 and max of integer dtype
- handling of label data
  - if label data has channel dimension, this will be squeezed, to prevent error
  - if label data is not specified, empty tensor will be returned rather than None, for compatibility with pytorch dataloader
- code changes in master from April 20 are ignored (overwritten, state corresponds to master from April 16, with above modifications)

### CODE REVIEW

**Comment by jakobtroidl** [comment]
Thanks @gbhuang! I also added the new normalization parameters to the README.
